### PR TITLE
Fix duplicate key

### PR DESCRIPTION
**Pull request by saltcod**
"Issues with Apis" and "Abuse report" have the same value:  "Problem" 
So both get selected when one of them is selected.  

CleanShot 2023-08-25 at 13 22 08@2x

### CODE REVIEW

File: studio/components/interfaces/Support/Support.constants.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const CATEGORY_OPTIONS = [
     query: undefined,
   },
   {
-    value: 'Problem',
+    value: 'Abuse',
```
Comment (joshenlim) [inline studio/components/interfaces/Support/Support.constants.ts:39]: before we get this in, just need to double check on API's end whether this is a valid category

Comment (joshenlim) [reply studio/components/interfaces/Support/Support.constants.ts:39]: yeah these are the available support categories so this change will send the support ticket to feedback instead

image

Comment (joshenlim) [reply studio/components/interfaces/Support/Support.constants.ts:39]: while you're on this PR @saltcod would you mind just changing the values to using `SupportCategories` from shared-types? will help us prevent any sort of deviance between FE and API

but yeah these comments are separate to the issue that you're fixing in this PR, may need a different solution

**Review by joshenlim — CHANGES_REQUESTED** [review]
Left some comments but TLDR, 'Abuse' as a category is not supported and will not be funnelled into the support flow

File: studio/components/interfaces/Support/SupportForm.tsx [hunk 2]
```diff
@@ -282,6 +282,7 @@ const SupportForm = ({ setSentCategory }: SupportFormProps) => {
               subject: subject ?? initialValues.subject,
               category: selectedCategoryFromUrl?.value ?? initialValues.category,
               message: message ?? initialValues.message,
+              severity: initialValues.severity,
```
Comment (saltcod) [inline studio/components/interfaces/Support/SupportForm.tsx:285]: Issue: 
https://supabase.slack.com/archives/C0161K73J1J/p1693412541497319?thread_ts=1693404721.979579&cid=C0161K73J1J

**Comment by saltcod** [comment]
Depends on
https://github.com/supabase/shared-types/pull/46

**Comment by joshenlim** [comment]
lgtm! tested every category on the preview - works :)

**Review by joshenlim — APPROVED** [review]

**Pull request merged**